frontend/src/shared/components/StatusToolTip.tsx

```diff
@@ -1,82 +1,206 @@
     {
-      field: "report",
-      headerName: "Report",
-      wrapText: true,
-      autoHeight: true,
-      flex: 2,
-      cellStyle: { wordBreak: "break-word" },
+      status: "Sample type abbreviation not resolved",
+      meaning:
+        "Metadata required for CMO Sample Name generation is missing/ incomplete",
+      actionItem: "PMs update metadata in SMILE PM dashboard",
+      responsibleParty: "PMs",
     },
-  ];
+  "recipe missing": {
+    status: "Recipe Missing",
+    meaning: "IGOComplete checkbox was not ticked in LIMS",
+    actionItem:
+      "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or " +
+      "clicking 'Mark delivery' button",
+    responsibleParty: "IGO",
+  },
+  "normalizedPatientId missing from 'cmoSampleIdFields'": {
+    status: "NormalizedPatientId Missing",
+    meaning: "IGOComplete checkbox was not ticked in LIMS",
+    actionItem:
+      "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or " +
+      "clicking 'Mark delivery' button",
+    responsibleParty: "IGO",
+  },
+  "specimenType (sampleClass) invalid": {
+    status: "Specimen Type (Sample Class) Invalid",
+    meaning:
+      "The value for the IGO Specimen Pype (SMILE Sample Class) is not recognized",
+    actionItem: "PMs update Specimen Type in SMILE PM dashboard",
+    responsibleParty: "PMs",
+  },
+  "sampleType missing from 'cmoSampleIdFields'": {
+    status: "Sample Type missing from 'cmoSampleIdFields'",
+    meaning:
+      "The label generator will fall back onto this field if the IGO Specimen Type (SMILE Sample Class) is not " +
+      "missing or is not a recognized value. This field is auto generated by IGO as part of the sample metadata",
+    actionItem: "PMs update Specimen Type in SMILE PM dashboard",
+    responsibleParty: "PMs",
+  },
+  "sampleType invalid 'sampleType' from 'cmoSampleIdFields'": {
+    status: "Sample Type Invalid in 'cmoSampleIdFields'",
+    meaning:
+      "The label generator will fall back onto this field if the IGO Specimen Type (SMILE Sample Class) is not " +
+      "missing or is not a recognized value. This field is auto generated by IGO as part of the sample metadata",
+    actionItem: "PMs update Specimen Type in SMILE PM dashboard",
+    responsibleParty: "PMs",
+  },
+  "investigatorSampleId missing": {
+    status: "InvestigatorSampleId Missing",
+    meaning:
+      "InvestigatorSampleId field does not have a value assigned in LIMS",
+    actionItem: "PMs update InvestigatorSampleId in SMILE PM dashboard",
+    responsibleParty: "PMs",
+  },
+};
+
+/**
+ * AG Grid data to use when a Sample is missing a corresponding Status in the database.
+ */
+const MISSING_SAMPLE_STATUS: SampleStatusItem[] = [
+  {
+    status: "Data error",
+    meaning: "Validation status is missing from this sample",
+    actionItem: "PMs contact the SMILE team",
+    responsibleParty: "SMILE",
+  },
+];
```

Comment:
I'm wondering if it makes more sense to store this info in something like a tab delimited file or a json that's loaded on startup that lives alongside the `application.properties` - just a thought that might make it a little easier to add new mappings on a whim without having to make changes to the code

Reply:
Actually yes, as I'm thinking about it more I think that would be best. Let's do a JSON file. 

Example JSON:
```
[
  {
    "error": "normalizedPatientId missing from 'cmoSampleIdFields'",
    "status": "NormalizedPatientId Missing",
    "meaning": "IGOComplete checkbox was not ticked in LIMS",
    "actionItem": "PMs contact IGO to fix and redeliver, let SMILE know once FASTQs are available by emailing them or clicking 'Mark delivery' button",
    "responsibleParty": "IGO"
  }
]
```